src/compat/index.js

```diff
@@ -7,10 +7,11 @@ const Responder = require('./responder')
 const Querier = require('./querier')
 
 class GoMulticastDNS extends EE {
-  constructor (peerInfo) {
+  constructor (peerId, multiaddrs) {
```

Comment:
Make this an object?
```suggestion
  constructor ({ peerId, multiaddrs }) {
```